cmd/client/main.go

```diff
@@ -87,6 +89,16 @@ func withArgs(args []string, errorf errorFn) {
 	rootCmd.AddCommand(reportCmd(rootArgs, errorf))
 	rootCmd.AddCommand(quotaCmd(rootArgs, errorf))
 
+	// The tracer will keep track of spans in memory and print them to stdout when the client exits.
+	recorder := tracer.NewInMemoryRecorder()
+	defer func() {
+		for _, span := range recorder.GetSpans() {
+			//fmt.Printf("%v\n", span)
+			glog.Infof("%v\n", span)
```

Comment:
We're using fmt.Printf for the main output of a command, and glog only for the error output, so we should be consistent here and use fmt.Printf.

Separately, we can argue whether the way the mixc and mixs handle output is desirable or not, but let's be consistent in the meantime at least.